main/lsp/LSPTypechecker.cc

```diff
@@ -641,51 +639,14 @@ LSPTypechecker::SlowPathResult LSPTypechecker::runSlowPath(LSPFileUpdates &updat
     }
 }
 
-void LSPTypechecker::cacheOpenFiles(absl::Span<const ast::ParsedFile> indexed, std::vector<core::FileRef> openFiles,
-                                    WorkerPool &workers) {
-    if (openFiles.empty()) {
-        return;
-    }
-
+void LSPTypechecker::cacheOpenFiles(absl::Span<const ast::ParsedFile> indexed,
```

Comment:
It might be worth calling this cacheUpdatedFiles, because these files are not necessarily open; they’re could be watchman-updated files.